src/call/mod.rs

```diff
@@ -71,28 +66,41 @@ macro_rules! status_codes {
                 )
             }
         }
+
+        impl Display for RpcStatusCode {
```

Comment:
What's the difference?

Reply:
The numeric value of the error code isn't included  and a human-readable version of the status code symbol is given (i.e. lowercase and underscores replaced with spaces). E.g. Instead of giving "15-DATA_LOSS", it gives "data loss". The former is great for the Debug trait implementation, but IMHO could be improved when being presented to a user, which is the intention of the Display implementation.